### PR TITLE
Adding team members' names and GitHub usernames

### PR DESCRIPTION
**Pull request by mariasam16**
Please review and add your usernames

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,13 @@
 <h1> <img src ="logo4.jpg" alt="Logo"></h1>
+
+Team members:
+
+1. Maria Alsamaien (mariasam16)
+2. Susmita Ghosh ()
```
Comment (Susmita-Rani-Ghosh) [inline README.md:6]: My user name is Susmita-Rani-Ghosh

Comment (mariasam16) [reply README.md]: Added!

File: README.md [hunk 2]
```diff
@@ -1,4 +1,13 @@
 <h1> <img src ="logo4.jpg" alt="Logo"></h1>
+
+Team members:
+
+1. Maria Alsamaien (mariasam16)
+2. Susmita Ghosh (Susmita-Rani-Ghosh)
+3. Geovanni Tinoco ()
```
[the remark this answers is not in the document]
Comment (mariasam16) [reply README.md]: Added!

File: README.md [hunk 3]
```diff
@@ -1,4 +1,13 @@
 <h1> <img src ="logo4.jpg" alt="Logo"></h1>
+
+Team members:
+
+1. Maria Alsamaien (mariasam16)
+2. Susmita Ghosh (Susmita-Rani-Ghosh)
+3. Geovanni Tinoco (GeoT82)
+4. Abdullahi (AbdullahiAyantayo)
+5. Mohammad ()
```
Comment (mhazadegan) [inline README.md:9]: mhazadegan

Comment (mariasam16) [reply README.md]: Added!

**Pull request merged**